lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -101,6 +102,22 @@ llvm::Error setTensorType(const caffe2::QTensorProto &in, Tensor *T) {
     RETURN_ERR("Only uint8 and int32 qtensors are supported");
   }
 }
+
+Storage *createPlaceHolderFromWeightTensor(
+    PlaceholderBindings *bindings, Function &f, Tensor *weights,
+    const std::string &name, bool moveOrCopy, bool isQuantized,
```

Comment:
Consider renaming `moveOrCopy` to `allowMove` or something like that. When `moveOrCopy` is `true`, it is not clear from the name what that means.